app/actors/hyrax/actors/create_with_remote_files_actor.rb

```diff
@@ -68,7 +68,7 @@ def create_file_from_url(env, uri, file_name, auth_header = {})
           ::FileSet.new(import_url: import_url, label: file_name) do |fs|
             actor = Hyrax::Actors::FileSetActor.new(fs, env.user)
             actor.create_metadata(visibility: env.curation_concern.visibility)
-            actor.attach_to_work(env.curation_concern)
+            env.curation_concern = actor.attach_to_work(env.curation_concern)
```

Comment:
i'm worried about this, because i've seen nasty failures outside the actor stack when trying to change the referent of `env.curation_concern`.

Are there any end-to-end tests of the actor stack that attach a remote file (and exercise this code)? I think i'd want to see something like that in `spec/features/actor_stack_spec.rb` and/or something that calls this code through a controller before moving forward with this.

I think, e.g., a `actor.create(env); env.curation_concern.persisted?` will be false with this, and lots of dependent code will fail as a result.